src/water.rs

```diff
@@ -7,7 +7,7 @@ use material::*;
 
 pub const WATER_SIZE: u32 = 256;
 pub const WATER_HALF_SIZE: f32 = WATER_SIZE as f32 / 2.0;
-pub const WATER_GRID_SIZE: u32 = 6;
+pub const DEFAULT_WATER_GRID_SIZE: u32 = 6;
```

Comment:
To avoid a breaking change, lets keep the old name for now.  For the next bevy release, we can change it.  I don't want to break code.  The other changes should be fine in a "minor" release.

Reply:
Ah, good point, I restored the constant name.

Reply:
Thanks.  Merged and published as `0.15.2`.